docs/products/kafka/howto/enable-governance.md

```diff
@@ -3,34 +3,36 @@ title: Enable governance for Aiven for Apache Kafka®
 sidebar_label: Enable governance
 limited: true
 ---
-import ConsoleLabel from "@site/src/components/non-swizzled/ConsoleIcons"
+
 import RelatedPages from "@site/src/components/non-swizzled/RelatedPages";
+import ConsoleLabel from "@site/src/components/non-swizzled/ConsoleIcons"
 
-Enable governance in Aiven for Apache Kafka® to establish a secure and compliant framework to manage your Aiven for Apache Kafka clusters efficiently.
+Enable governance in Aiven for Apache Kafka® to create a secure and compliant framework to manage your Aiven for Apache Kafka services efficiently.
+
+By default, governance applies at the organization level. Admins can manually select
+specific services to govern through the Apache Kafka governance settings
 
 ## Impact of enabling governance
 
 - **Existing topics**:
-  - The selected default group is assigned as the owner of all existing Apache Kafka
-    resources.
+  - The default group owns all existing Apache Kafka resources.
   - Ownership details for Apache Kafka resources are visible in the
     [Apache Kafka topic catalog](/docs/products/kafka/concepts/topic-catalog-overview).
-  - Users from different groups can still claim ownership of individual resources.
+  - Users in different groups can request ownership of individual resources.
 
 - **Topic creation workflow**:
-  - There is no impact on existing topics.
+  - Existing topics remain unaffected
   - You can continue to [create topics](/docs/products/kafka/howto/create-topic) in your
-    Aiven for Apache Kafka service. Governance in Aiven for Apache Kafka introduces an
-    additional request-and-approval process to claim ownership of topics from the
-    Apache Kafka topic catalog.
+    Aiven for Apache Kafka service. Governance adds a request-and-approval process for
+    claiming topic ownership through the Apache Kafka topic catalog.
   - All topics align with your organization's data management policies.
 
 ## Prerequisites
 
 - This is a [limited availability feature](/docs/platform/concepts/beta_services). To try
   it out, contact the sales team at [sales@aiven.io](mailto:sales@aiven.io).
-- You must be an
-  [organization admin](/docs/platform/concepts/permissions#organization-roles-and-permissions)
+- Ensure you have
+  [super admin permissions](/docs/platform/howto/manage-permissions#make-users-super-admin)
```

Comment:
Is super admin access still needed?

Reply:
I am checking this with the team. So far, they have mentioned that super admin access is needed. There is an open ticket to test this, and I will update based on the results.